extensions/bicep/bicep.tests.ps1

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+BeforeDiscovery {
+    $foundBicep = if ($null -ne (Get-Command bicep -ErrorAction Ignore)) {
+
+        $true
+    } else {
+        $false
+    }
```

Comment:
```suggestion
    $foundBicep = if (Get-Command bicep -ErrorAction Ignore) {
        $true
    } else {
        $false
    }
```

Reply:
I tend to find negative named variables harder to read